workspaces/copilot/.changeset/sixty-numbers-count.md

```diff
@@ -0,0 +1,9 @@
+---
+'@backstage-community/plugin-copilot-backend': minor
+---
+
+Various performance improvements for larger organizations
+
+- Only gather metrics from teams that have 5 or more members
```

Comment:
Can we include a link to the docs where it says it won't provide details for teams of less than 5? Can we also update the plugin's `README` with this information as well?

Reply:
Added